src/main/java/com/interview/library/service/BookService.java

```diff
@@ -50,6 +51,17 @@ public Book save(Book book) {
         return bookRepository.findAll(pageable);
     }
 
+    /**
+     * Get all books its price is lower than <i>priceThreshold</i> parameter.
```

Comment:
`Get all books its price is lower` -> `Get all books where the price is lower`

Reply:
Changed (modified it a bit since it contained more errors)

Reply:
ok